### PR TITLE
Fix create new request process manually

### PR DESCRIPTION
**Pull request by AmiAhm**
Fixes "Your project must not create a new Request process manually" produced by Symfony Insight

### CODE REVIEW

**Comment by MathiasWahl** [comment]
Symfony insight error message is gone 😍

**Comment by MathiasWahl** [comment]
Closes #1379

**Review by MathiasWahl — APPROVED** [review]
I think this is good to merge, but please be aware that some tests have been altered to pass. The rationale for allowing this is that the fixtures have been altered since the creation of the tests, altering some fixture entities' ids and thus making the tests fail.
I have tried to confirm that the intended ids are restored, but this should also be verified by someone else before merging 😄

File: tests/AppBundle/Availability/AvailabilityFunctionalTest.php [hunk 1]
```diff
@@ -258,7 +258,7 @@ public function teamLeaderUrlProvider()
             array('/kontrollpanel/hovedstyret/oppdater'),
 
             array('/kontrollpanel/opptakadmin/teaminteresse'),
-            array('/kontrollpanel/opptakadmin/teaminteresse?department=2&semester=1'),
+            array('/kontrollpanel/opptakadmin/teaminteresse?department=1&semester=1'),
```
Comment (MathiasWahl) [inline tests/AppBundle/Availability/AvailabilityFunctionalTest.php:261]: Here is one of the tests that had the id changed. I suspect this was a typo and was supposed to reference department 1, as the TeamLeader client in question was also a member of department 1.

File: tests/AppBundle/Controller/MailingListControllerTest.php [hunk 2]
```diff
@@ -76,7 +78,7 @@ private function generateListCountChars(Client $client, string $type)
     {
         $crawler = $this->goTo('/kontrollpanel/epostlister', $client);
         $form = $crawler->selectButton('Generer')->form();
-        $form['generate_mailing_list[semester]'] = 2;
+        $form['generate_mailing_list[semester]'] = 1;
```
Comment (MathiasWahl) [inline tests/AppBundle/Controller/MailingListControllerTest.php:81]: Here is the other test that had its id changed. I suspect that it is supposed to reference the current semester, which in an earlier state had id = 2. Currently, semester with id = 2 references the previous semester, and that semester does not have any entries in the Assistant mailing list (whereas the current semester has 2)

File: tests/AppBundle/Availability/AvailabilityFunctionalTest.php (hunk 1 above)
Comment (MathiasWahl) [reply tests/AppBundle/Availability/AvailabilityFunctionalTest.php:261]: ⚠️ THIS SHOLD BE APPROVED BEFORE MERGE ⚠️

File: tests/AppBundle/Controller/MailingListControllerTest.php (hunk 2 above)
Comment (MathiasWahl) [reply tests/AppBundle/Controller/MailingListControllerTest.php:81]: ⚠️ THIS SHOULD BE APPROVED BEFORE MERGE ⚠️

File: tests/AppBundle/Availability/AvailabilityFunctionalTest.php (hunk 1 above)
Comment (MathiasWahl) [reply tests/AppBundle/Availability/AvailabilityFunctionalTest.php:261]: Here you can see the typo I think has occurred. In the top line, the semester with id = 4 is changed to `department=1&semester=1`. I the bottom line, the same semester is changed to `department=2&semester=1`, which causes the test to fail. 
Screenshot 2020-09-24 at 12 13 52

The change came with `Fjern department fra semester`, #1048 by @sigtot, so perhaps he has some insight into this 😄

Comment (sigtot) [reply tests/AppBundle/Availability/AvailabilityFunctionalTest.php:261]: Lgtm

**Review by sigtot — APPROVED** [review]

File: tests/AppBundle/Controller/MailingListControllerTest.php (hunk 2 above)
Comment (MathiasWahl) [reply tests/AppBundle/Controller/MailingListControllerTest.php:81]: The mentioned change was also introduced with `Fjern department fra semester`, #1048 by @sigtot :

Before. note, here the current semester will have id = 2, as it is the second entity to be persisted to the db:
Screenshot 2020-09-24 at 12 23 59

After. Here, the current semester is persisted first and will have id = 1:
Screenshot 2020-09-24 at 12 24 12

**Pull request merged**